src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,39 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ProfileServlet extends HttpServlet {
+
+    /**
+     * Store class that gives access to Users.
+     */
+    private UserStore userStore;
+
+
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        setUserStore(UserStore.getInstance());
+    }
+
+    /**
+     * Sets the UserStore used by this servlet. This function provides a common setup method for use
+     * by the test framework or the servlet's init() function.
+     */
+    void setUserStore(UserStore userStore) {
+        this.userStore = userStore;
+    }
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
+        String username = (String) request.getSession().getAttribute("user");
+        request.getRequestDispatcher("/WEB-INF/view/profile.jsp").forward(request, response);
+    }
+}
```

Comment:
Can we add the doPost method as well? along with the documentation for it. Even if its empty. Since we are overriding HttpServlet, it is good to override all the methods we are interested in so we dont forget in the future.

Reply:
Got it. I will add doPost method as well.